extensions/public-rest-api/src/main/java/org/eclipse/dataspaceconnector/demo/api/rest/IonDemoApiExtension.java

```diff
@@ -1,6 +1,7 @@
 package org.eclipse.dataspaceconnector.demo.api.rest;
 
 import org.eclipse.dataspaceconnector.catalog.spi.QueryEngine;
+import org.eclipse.dataspaceconnector.spi.contract.negotiation.ConsumerContractNegotiationManager;
```

Comment:
Same remark for the extension name

Reply:
if necessary, we can raise this in the EDC repo